catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/SecurityAttributes.java

```diff
@@ -74,6 +74,56 @@
                 false /* tokenized */,
                 true /* multivalued */,
                 BasicTypes.STRING_TYPE));
+
+        /* Experimental Attributes, subject to change*/
+        DESCRIPTORS.add(new AttributeDescriptorImpl(METADATA_ORIGINATOR_CLASSIFICATION,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(METADATA_CLASSIFICATION,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(METADATA_DISSEMINATION,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
```

Comment:
The MGMP specification limits the occurrences of dissemination and releasability to one string delimited by slashes for each country.  In general, I think that these attributes (`DISSEMINATION` and `RELEASABILITY`) should be multivalued.